test/Renci.SshNet.IntegrationTests/TestsFixtures/InfrastructureFixture.cs

```diff
@@ -28,49 +28,44 @@ public static InfrastructureFixture Instance
 
         public ushort SshServerPort { get; set; }
 
-        public SshUser AdminUser = new SshUser("sshnetadm", "ssh4ever");
+        public SshUser AdminUser { get; } = new SshUser("sshnetadm", "ssh4ever");
 
-        public SshUser User = new SshUser("sshnet", "ssh4ever");
+        public SshUser User { get; } = new SshUser("sshnet", "ssh4ever");
 
         public async Task InitializeAsync()
         {
             _sshServerImage = new ImageFromDockerfileBuilder()
                 .WithName("renci-ssh-tests-server-image")
                 .WithDockerfileDirectory(CommonDirectoryPath.GetSolutionDirectory(), Path.Combine("test", "Renci.SshNet.IntegrationTests"))
                 .WithDockerfile("Dockerfile")
-                .WithDeleteIfExists(true)
-                
+                .WithDeleteIfExists(deleteIfExists: true)
```

Comment:
I am all for named parameters to provide context, but in cases like this and in `ConfigureAwait(false)` (which is practically a canonical form) it is just noise. I would prefer this analyzer to be disabled